OpenRA.Mods.Common/Traits/HitShape.cs

```diff
@@ -20,6 +21,9 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Shape of actor for targeting and damage calculations.")]
 	public class HitShapeInfo : ConditionalTraitInfo, Requires<BodyOrientationInfo>
 	{
+		[Desc("Name of turret this shape is linked to. Leave empty to link shape to body.")]
+		public readonly string LinkedTurret = null;
```

Comment:
We call this just `Turret` in `Armament` and `With*Turret`, so can we please rename this to match?